### PR TITLE
chore(colors): remove duplicate colors

### LINKED ISSUE #7018: Duplicate colors in the color selector
There appear to be duplicate colors in the color selector.

### How to test
- Use bhima_test
- Go to Adminstration > Debtor Group Management
- Edit one of the groups
- Click on the color selector and observe the duplicate colors
   - For instance, type in 'Jaune'

### PR DESCRIPTION
**Pull request by jniles**
This commit removes duplicate colors from the ColorService.

Closes #7018

### CODE REVIEW

File: client/src/js/services/color.js [hunk 1]
```diff
@@ -6,17 +6,6 @@ ColorService.$inject = [
 function ColorService($translate) {
   const service = this;
   service.list = [
-    { name : $translate.instant('COLORS.AQUA'), value : '#00ffff' },
```
Comment (jmcameron) [inline client/src/js/services/color.js:9]: Thank you!

**Review by jmcameron — CHANGES_REQUESTED** [review]
I noticed this problem while converting the E2E tests.   Thanks for working on it!

The changes mostly look good.  

I tested it using the bhima_test DB.  I noticed that the colors that are from i18n COLORS.* are capitalized.  Could you please modify the .json file and uncapitalize them.  Also I think GRAY is a duplicate that could be deleted.

Thanks!

**Review by jmcameron — CHANGES_REQUESTED** [review]
Thanks for the changes.  I hate to be picky, but could I ask you to delete the duplicate 'gray' color and sort the list of colors alphabetically.  And why do we have translations for some colors and not others (legacy?).

**Comment by jniles** [comment]
@jmcameron this is ready to go 👍

**Comment by jniles** [comment]
Looks like MySQL failed to initialize - I'll restart it.

bors r+

**Pull request merged**